setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='bamboolean',
-    version='0.1.0',
+    version='0.1.2',
```

Comment:
inb4: why you bump version to 0.1.2 instead of 0.1.1?

I made a mistake while uploading 0.1.1 version, removed it, and wanted to re-upload. Apparently reuploading is impossible in PyPI, thus, i needed to jump over 0.1.1 and move to another one. Related issue:
https://github.com/pypa/packaging-problems/issues/74